content/php-client/resources/target-market-settings/measurement-families.md

```diff
@@ -0,0 +1,116 @@
+### Measurement family
+
+#### Get a list of measurement families
+::: php-client-availability versions=6.0 editions=CE,EE
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
```

Comment:
```suggestion
 * Returns an array as following:
```
?

Reply:
I wouldn't mind but it's written exactly like this in all the other phpclient pages.